build/commands/scripts/commands.js

```diff
@@ -30,6 +30,18 @@ function commaSeparatedList(value, dummyPrevious) {
   return value.split(',')
 }
 
+// Use this wrapper function instead of JavaScript's parseInt() with option()
+// when defining integer optional parameters, or the default value might get
+// passed as well into the radix parameter of parseInt(), causing wrong results.
+// https://github.com/brave/brave-browser/issues/13724
+function parseInteger(string) {
```

Comment:
Does `commander` supply the configured default value here as the `string` parameter, if none is specified on the command line (e.g. `4` in the `test_launcher_jobs` case)?

Reply:
Yes, it does.